### PR TITLE
proper loading logic

### PR DESCRIPTION
**Pull request by andreasfrisch**
closes dfds/cloudplatform/issues/2068

# Additional Review Notes
Also fixed the text in a prompt

### CODE REVIEW

File: src/src/hooks/ECRRepositories.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import { useEffect, useState } from "react";
 import { useSelfServiceRequest } from "./SelfServiceApi";
+import { set } from "date-fns";
```
Comment (SEQUOIIA) [inline src/src/hooks/ECRRepositories.js:3]: Wait where is this being used?

**Review by SEQUOIIA — CHANGES_REQUESTED** [review]
One comment, that aside proper loading logic sounds nice

File: src/src/hooks/ECRRepositories.js (hunk 1 above)
Comment (andreasfrisch) [reply src/src/hooks/ECRRepositories.js]: Which?
I see nothing ... 😄

**Review by SEQUOIIA — APPROVED** [review]
I see nothing

**Pull request merged**